packages/tldraw/src/lib/ui/components/primitives/TldrawUiKbd.tsx

```diff
@@ -5,12 +5,13 @@ import { kbd } from '../../kbd-utils'
 /** @public */
 export interface TLUiKbdProps {
 	children: string
+	visibleOnMobileLayout?: boolean
 }
 
 /** @public */
-export function TldrawUiKbd({ children }: TLUiKbdProps) {
+export function TldrawUiKbd({ children, visibleOnMobileLayout = false }: TLUiKbdProps) {
 	const breakpoint = useBreakpoint()
-	if (breakpoint < PORTRAIT_BREAKPOINT.MOBILE) return null
+	if (!visibleOnMobileLayout && breakpoint < PORTRAIT_BREAKPOINT.MOBILE) return null
```

Comment:
Normally we hide keyboard shortcuts on small screens (ie: mobile screens).

But if you're looking at the keyboard shortcuts menu, we should *always* show the keyboard shortcut. So I added a prop that lets you override the auto-hiding. This is a pretty common case you can get to by shrinking your browser window when you have many windows open.

I could understand the viewpoint that this should decision should come from higher up the tree, but I'm also happy to keep behaviour how it currently is - seems fine to me.

Reply:
hah, yeah this feels funny. i think it's prbly better to leave this one be? feels like it's the wrong abstraction for this and hard to get into this case anyway.